tts/Object.def.lua

```diff
@@ -517,6 +522,10 @@ function Object.removeButton(index) end
 ---@return true
 function Object.scale(scale) end
 
+
+---@param notes string
+---@return boolean
```

Comment:
Can be changed to:
```
---@return true
```
as it never returns `false`.

Reply:
Done